wandb/sdk/lib/stdin_timeout.py

```diff
@@ -0,0 +1,66 @@
+import sys
```

Comment:
New files in lib should get some type annotations..  Add your file to mypy.ini and annotate what makes sense, dont need everything, but it would be nice if it was tested.  For example we have one here:

```
[mypy-wandb.sdk.lib.telemetry]
disallow_untyped_defs = True
disallow_untyped_calls = True
```

Reply:
I've annotations in the stdin_timeout.py file